backend/api/migrations/0041_rename_factory_model.py

```diff
@@ -1,16 +1,94 @@
-# Generated by Django 2.2.13 on 2021-10-20 12:37
+# Generated by Django 2.2.13 on 2022-04-23 22:24
 
-from django.db import migrations
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.db.models.manager
+import uuid
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('api', '0040_rename_factory_type'),
     ]
 
     operations = [
-        migrations.RunSQL(
-            "ALTER TABLE api_factory RENAME TO api_ohshownevent;"
```

Comment:
@EagleC0318 for example, instead of migrating data by sql injection, I prefer 1) create the new model 2) inject seed data for this new model

Reply:
The following is how we should "create the new model".